example/properties_directory/structural/inchi.json

```diff
@@ -0,0 +1,4 @@
+{
+    "name": "inchi",
+    "value": "1S/Si2/c1-2"
```

Comment:
Is this a real-world inchi? Maybe let's use something more realistic?

Reply:
yes it actually is the real world InChI for Si2, at least it is the one the at is calculated from openbabel. I believe it prints out. I can use another material, like one from the examples in the [hash generation confluence page with inchi examples](https://exabyte.atlassian.net/wiki/spaces/SCI/pages/1836548137/Hash+Creation+for+Molecular+Systems)